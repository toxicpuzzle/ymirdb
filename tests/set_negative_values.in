SET a1b -100 1 001 
SET b2c 100 1 -2000
SET c3d 10 20 -100
LIST ENTRIES
bye