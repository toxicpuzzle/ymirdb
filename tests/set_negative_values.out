> ok

> ok

> ok

> c3d [10 20 -100]
b2c [100 1 -2000]
a1b [-100 1 1]

> bye
